addons/mail/static/src/new/discuss/discuss.xml

```diff
@@ -64,22 +64,65 @@
+        <div t-if="!thread and (!env.isSmall or store.discuss.activeMobileNavbarTabId === 'mailbox')" class="o-mail-discuss-no-thread d-flex flex-grow-1 align-items-center justify-content-center w-100 bg-view">
             <h4 class="text-muted"><b><i>No conversation selected.</i></b></h4>
         </div>
+        <t t-if="env.isSmall and ['chat', 'channel'].includes(store.discuss.activeMobileNavbarTabId)">
+            <button t-if="store.discuss.activeMobileNavbarTabId === 'chat'" class="w-100 p-2 btn btn-secondary border-bottom bg-light">
+                Start a conversation
+            </button>
+            <button t-if="store.discuss.activeMobileNavbarTabId === 'channel'" class="w-100 p-2 btn btn-secondary border-bottom bg-light">
+                New Channel
+            </button>
+            <div class="d-flex flex-column overflow-auto flex-grow-1 w-100">
+                <t t-foreach="displayedPreviews" t-as="preview" t-key="preview.id">
+                    <NotificationItem
+                        body="preview.mostRecentMsg?.inlineBody"
+                        dateTime="preview.mostRecentMsg?.dateTime"
+                        displayName="preview.displayName"
+                        iconSrc="preview.imgUrl"
+                        hasMarkAsReadButton="preview.isUnread"
+                        isLast="preview_last === 0"
+                        onClick="(isMarkAsRead) => isMarkAsRead ? threadService.markAsRead(preview) : threadService.open(preview)"
+                    >
+                        <t t-set-slot="icon">
+                            <PartnerImStatus t-if="preview.type === 'chat'" className="'position-absolute bottom-0 end-0'" partner="store.personas[createLocalId('partner', preview.chatPartnerId)]"/>
+                        </t>
+                        <t t-if="preview.mostRecentMsg" t-set-slot="body-icon">
+                            <t t-if="preview.mostRecentMsg.isSelfAuthored">
+                                <i class="fa fa-mail-reply me-1"/>You
+                            </t>
+                            <t t-else="">
+                                <t t-esc="preview.mostRecentMsg.author.name"/>
+                            </t>:
+                        </t>
+                    </NotificationItem>
+                </t>
+            </div>
+        </t>
+        <t t-if="env.isSmall">
+            <t t-set="tabs" t-value="[
+            {icon: 'fa fa-inbox',id: 'mailbox',label: 'Mailboxes'},
+            {icon: 'fa fa-user',id: 'chat',label: 'Chat'},
+            {icon: 'fa fa-users',id: 'channel',label: 'Channel'}
+            ]"/>
+            <div class="o-mail-discuss-mobile-nav-bar d-flex flex-shrink-0 border-top bg-view shadow-lg w-100">
+                <t t-foreach="tabs" t-as="tab" t-key="tab.id"  t-call="mail.discuss_mobile_nav_tab"/>
+            </div>
```

Comment:
There's similar code in `messaging_menu.xml`.
Would be nice to share the code between both of them.

Actually discuss in mobile is messaging menu as full screen. Could be worth almost reusing `MessagingMenu` component?